development/tests.py

```diff
@@ -42,3 +52,90 @@ def test_create_match(
         self.assertEqual(match.bot_2, bot_two)
         self.assertEqual(match.score_p_1, scr1)
         self.assertEqual(match.score_p_2, scr2)
+
+
+def mocked_requests_get(*args, **kwargs):
+    key = kwargs['key']
+    data = kwargs['data']
+
+    class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.status_code = status_code
+
+        def json(self):
+            return self.json_data
+
+    return MockResponse({key: data}, 200)
+
+
+class TestBotHandler(TestCase):
```

Comment:
I think you are missing the scenario where you are catching an exception

Reply:
for the next sprint!